src/main/resources/com/adaptavist/tm4j/jenkins/extensions/postbuildactions/TestResultPublisher/help-customTestCycle.html

```diff
@@ -1,6 +1,5 @@
 <div>
-    <strong>Zephyr Scale Cloud only. If sent to Zephyr Scale Server/DC,
-        the values will be ignored.</strong>
+    <strong>If sent to Zephyr Scale Server/DC, version 8.7.5 is required</strong>
```

Comment:
Maybe we can say that it requires a version greater than `8.7.4`. I'm not sure that the next version would be `8.7.5`, it might be `8.8.0`. Also sometimes, it's rare but it happens, we need to "retire" a version that was released with problems. Let's say that we release version 8.7.5 and we need to remove that version from the Atlassian marketplace, we would need to release a new version of the Jenkins plugin to fix the message.

Reply:
:+1: 
I've already sent a message to David to know if we wait for the next release to merge this pull request, when I'll have the answer and the definitive version I'll update the message.